assets/agenda/components/AgendaApp.jsx

```diff
@@ -86,7 +86,10 @@ class AgendaApp extends BaseApp {
 
     render() {
         if (this.state.initialLoad) {
-            return this.renderLoader();
+            return <div className="wire-articles__item-wrap col-12">
```

Comment:
The `renderLoader` is there to indicate to the user that the page/app is still loading. Therefor to display this message before loading has completed may confuse end users.

Instead you would need to check the user products, and if they're not admin, then show this alert